BlueprintUI/Sources/UIKit/PointerInteraction.swift

```diff
@@ -0,0 +1,309 @@
+//
+//  PointerInteraction.swift
+//  BlueprintUICommonControls
+//
+//  Created by Kyle Van Essen on 6/26/20.
+//
+
+import UIKit
+
+
+/// An `Element` which lets you add `UIPointerInteraction` interactions
+/// to the contained `Element`.
+///
+/// A `UIPointerInteraction` allows you to respond to hover events for buttons,
+/// text, icons, etc, when a user is using an iPad that has support for a pointer, usually from a trackpad.
+///
+/// - Note: On iOS versions before 13.4, this has no effect.
```

Comment:
Should we just make this available in 13.4+? If you're using it in your project a lot and target less than that, it's easy enough to add your own wrapper that does nothing on < 13.4. Seems a little weird to have that built in though.

Reply:
I don't think so. It's annoying to have to turd your code with these checks, when the end result you want is a no-op on earlier iOS versions anyway.